packages/ui-components/package.json

```diff
@@ -19,7 +19,8 @@
     "storybook": "start-storybook -s public -p 6006",
     "test": "yarn package:test --env=jsdom",
     "test:coverage": "yarn test --coverage",
-    "test:watch": "yarn test --watch"
+    "test:watch": "yarn test --watch",
+    "start-dev": "babel src --out-dir dist -w"
```

Comment:
At the risk of having too many cooks in the kitchen (too many reviewers), can we follow the convention we have and call this `build-dev:watch`?

Reply:
Also, this seems like a great candidate for a reusable script. See `package:build:js`, it is written once in the root package.json and then called by many packages. This also means we don't need to have babel as a dependency, it uses the standard package build script.

Reply:
Thanks @IgorNadj, I have pushed those changes.

Reply:
Much appreciated 🙏